assets/src/edit-story/elements/image/edit.js

```diff
@@ -40,22 +41,29 @@ const Element = styled.div`
   ${elementFillContent}
 `;
 
+// Opacity of the mask is reduced depending on the opacity assigned to the image.
 const FadedImg = styled.img`
   position: absolute;
-  opacity: 0.4;
+  opacity: ${({ opacity }) =>
+    opacity ? opacity * MEDIA_MASK_OPACITY : MEDIA_MASK_OPACITY};
   pointer-events: none;
   ${imageWithScale}
 `;
 
+// Opacity is adjusted so that the double image opacity would equal
+// the opacity assigned to the image.
 const CropImg = styled.img`
   position: absolute;
+  opacity: ${({ opacity }) =>
+    opacity ? 1 - (1 - opacity) / (1 - opacity * MEDIA_MASK_OPACITY) : null};
```

Comment:
wow! but does `null` work here as a CSS value?

Reply:
It should just dismiss the opacity if not set which should be OK. The alternative would be just setting it to `1`.